.github/RELEASE_CHECKLIST.md

```diff
@@ -44,12 +45,15 @@
    - https://hub.docker.com/r/ferretdb/ferretdb/tags
    - https://hub.docker.com/r/ferretdb/ferretdb-dev/tags
    - https://hub.docker.com/r/ferretdb/ferretdb-eval/tags
+   - https://hub.docker.com/r/ferretdb/ferretdb-eval-dev/tags
    - https://github.com/FerretDB/FerretDB/pkgs/container/ferretdb
    - https://github.com/FerretDB/FerretDB/pkgs/container/ferretdb-dev
    - https://github.com/FerretDB/FerretDB/pkgs/container/ferretdb-eval
+   - https://github.com/FerretDB/FerretDB/pkgs/container/ferretdb-eval-dev
    - https://quay.io/repository/ferretdb/ferretdb?tab=tags
    - https://quay.io/repository/ferretdb/ferretdb-dev?tab=tags
    - https://quay.io/repository/ferretdb/ferretdb-eval?tab=tags
+   - https://quay.io/repository/ferretdb/ferretdb-eval-dev?tab=tags
```

Comment:
Is there something we need to configure on quay to allow image access? Above returns `Session Expired` error for me.

Reply:
Amazing :hugs: thank you!